calcom/cal.com
# fix(platform): missing string literal opening

## What does this PR do?

This fixes an issue where the title is not rendered correctly and shows the code and not the translated string output.

Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is a UI/UX design document, please, share it here.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Go to https://app.cal.com/availability/<id> and check tab title to make sure it is correctly rendering.

## Mandatory Tasks
